src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -60,23 +56,19 @@ private Helios() {
   }
 
   public static String getAccessToken(User user) {
-    return getAccessToken(user.getUserName(), user.getPassword());
+    return getAccessToken(user.getUserName());
   }
 
   public static void deleteAllTokens(User user) {
     String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.USERS);
 
     HttpDelete httpDelete =
         new HttpDelete(String.format("%s/%s/tokens", heliosGetTokenURL, user.getUserId()));
-    httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
-    httpDelete.setHeader("X-Wikia-Internal-Request", "0");
-
-    CloseableHttpResponse response = null;
-    try {
-      response = getDefaultClient().execute(httpDelete);
+    httpDelete.setHeader(THE_SCHWARTZ, Configuration.getCredentials().apiToken);
+    httpDelete.setHeader(X_WIKIA_INTERNAL_REQUEST, "0");
 
+    try (CloseableHttpResponse response = getDefaultClient().execute(httpDelete)) {
       PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
-
     } catch (ClientProtocolException e) {
       PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "CLIENT PROTOCOL EXCEPTION" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)